services/core-api/app/api/now_applications/resources/now_application_import_resource.py

```diff
@@ -65,5 +67,12 @@ def post(self, application_guid):
         if now_application_identity.now_application_id is not None:
             raise BadRequest('This record has already been imported.')
         application.save()
+        db.session.refresh(now_application_identity)
+        status = NOWApplicationStatus.find_by_now_application_status_code(
+            now_application_identity.now_application.now_application_status_code)
```

Comment:
now_application.now_application_status should return this. if it doesn't exist, can you add that relationship and use it here?

Reply:
Added